### ISSUE TITLE
scheduler marking shift+click 

### ISSUE THREAD

**Issue by blixten85** [Novik/ruTorrent#2630]
### What would you like changed about the web client?

As of today you have to click each pixel square in the scheduler in order to set it up for a speedlimit.

### Describe the solution you'd like and propose possible alternatives.

It would be much better if you could like shift+left click and then drag a "window" in order to cover way more squares in just one click. Just like you would do with a print screen feature where you only want to capture a certain field on the monitor/screen. 

### Provide information and resources about the environment hosting the web client.

Raspberry pi 5 

No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 12 (bookworm)
Release:        12
Codename:       bookworm

Linux raspberrypi 6.1.0-rpi7-rpi-2712 #1 SMP PREEMPT Debian 1:6.1.63-1+rpt1 (2023-11-24) aarch64 GNU/Linux

**Comment by stickz** [comment]
Yes, we can improve the scheduler plugin in version 4.3. (develop branch) The problem though is finding someone willing to do it.

**Comment by blixten85** [comment]
> Yes, we can improve the scheduler plugin in version 4.3. (develop branch) The problem though is finding someone willing to do it.

Have you found some yet? 
Easiest would be to just add boxes where time numbers can be added. It gotta be the least amount of work?
